libs/dh/wholesale/feature-settlement-reports/src/lib/tabs/dh-wholesale-settlements-reports-tabs-balance.component.html

```diff
@@ -52,13 +51,60 @@ <h3>
           "
         ></watt-dropdown>
       </watt-form-field>
-      <watt-button
-        [disabled]="!searchForm.valid"
-        *dhPermissionRequired="['settlements:manage']"
-        variant="secondary"
-        (click)="downloadClicked()"
-        >{{ t("download") }}</watt-button
-      >
     </form>
+    <watt-table
+      [dataSource]="dataSource"
+      [columns]="columns"
+      [resolveHeader]="translateHeader"
+      [selectable]="true"
+      description="batches"
+      sortBy="executionTime"
+      sortDirection="desc"
+    >
+      <!-- Grid Area Name -->
+      <ng-container *wattTableCell="columns.gridAreaName; let gridArea">
+        {{ gridArea.name }} ({{ gridArea.code }})
+      </ng-container>
+
+      <!-- Download -->
+      <ng-container *wattTableCell="columns.download; let gridArea">
+        <watt-button
```

Comment:
suggestion: instead of moving the button to the right in order to align with the column's header, let's use the `WattTableCellDirective`'s `header` input and set it to an empty string. This way the header of this column will be empty, thus no need of moving the button.